python/tach/cli.py

```diff
@@ -123,9 +123,9 @@ def print_unused_dependencies(
         + constraint_messages
     )
     print(
-        f"{BCOLORS.WARNING}\nRemove the unused dependencies from tach.yml, "
-        f"or consider running 'tach sync --prune' to update module configuration and "
-        f"eliminate all unused dependencies.\n{BCOLORS.ENDC}"
+        f"{BCOLORS.WARNING}\nAdd the unused dependencies from tach.yml, "
+        f"or consider running 'tach sync --add' to update module configuration and "
+        f"add all unused dependencies.\n{BCOLORS.ENDC}"
     )
```

Comment:
```suggestion
        f"{BCOLORS.WARNING}\nRemove the unused dependencies from tach.yml, "
        f"or consider running 'tach sync' to update module configuration and "
        f"remove all unused dependencies.\n{BCOLORS.ENDC}"
    )
```

Reply:
In this case, we are showing an error to the user because they have unused dependencies in their `tach.yml`, so our advice should still be to remove those dependencies.

The part that changes is that since `--prune` is now the default, we can just tell them to run `tach sync`